src/commands/account/create.js

```diff
@@ -14,7 +14,7 @@
  *
  */
 import { flags as flagParser } from '@oclif/command';
-import cryptography from '@liskhq/lisk-cryptography';
+import * as cryptography from '@liskhq/lisk-cryptography';
```

Comment:
Can't we have destructured `{}` import here, as we are just using a couple of functions?